backend/src/services/artist.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Artist, ArtistInterface } from "../models/artist.model";
+import { UpdateOptions } from "sequelize";
+
+export class ArtistService {
+  public async create(artist: ArtistInterface) {
+    const createdArtist: Artist = await Artist.create<Artist>(artist);
+    return { success: true, body: createdArtist };
+  }
+
+  public async show(artistId: number) {
+    const artist = await Artist.findByPk<Artist>(artistId, {
+      rejectOnEmpty: true,
+    });
+    return { success: true, artist: artist };
+  }
+
+  public async update(artistId: number, params: ArtistInterface) {
+    const update: UpdateOptions = {
+      where: { id: artistId },
+    };
+    await Artist.findByPk<Artist>(artistId, { rejectOnEmpty: true });
+    await Artist.update(params, update);
```

Comment:
@JDziewonski98 this syntax is interesting - findByPk then update on the same reference - does it work?

Reply:
I agree this syntax is weird, but it does work: notice you define the const update : UpdateOptions above, which has where: id: artistId. So even though you are calling the artist Class rather than an instance, it applies the update to the instance with that id. If this is confusing, and it might be, we can look into using a different syntax (saving the result of findByPk, updating that, then saving it.)

Reply:
I see, so findByPk is to ensure the mural at muralId exists before attempting to update it?